### PR TITLE
Roundstart Gameplay Tweak [Round start Evolution Changes and Survivor Rebalance] (RE-OPENED)

### PR DESCRIPTION
**Pull request by PalJohnson**
With permission from geeves i re-opened his MR and made changes with feedback from devs and players Bellow is the or
_**Shakes up roundstart gameplay for everyone ground side by implementing a bunch of wild changes:**_

Survivors now have role-specific skills, meaning one survivor is weaker, and need allies to cover their deficiencies.
More survivors spawn, with a minimum of 3(up from 2) and a maximum of 10 (up from 8) slots. Additionally, the calculation has been changed to 1 per 9 people (up from 1 per 11).
Added a PMC synth.
Survivor weapon pool received a rebalance pass and lore considerations
Survivors no longer have marine IFF.
Survivors now spawn with a medium rather than a large stack of metal.
Xeno castes can now only evolve after a certain amount of time has passed, generally, the more dangerous the caste, the longer it takes until they're available.
Xenos now gain "bonus evolution" when they gain evolution past their evolution max. This bonus evolution is carried on to their next xeno when they evolve, meaning staying T1 for a while doesn't waste your T2 evo.
Larva evo is no longer special, and works as all other xeno evo does.

**_Civilian Survivor Changes_**

- The civilian (colonist) survivor was added to the roster once again to help bring from love in the RP department and challenge. This survivor has the baseline stats (see bellow) and nothing else other than some cool clothing to compliment their role. The current clothing presets are janitor, bartender, colonist, and botanist. Being rather unimpressive in the skills department they will really need to use your wits to survive.

**_CLF Survivor_**

- Alongside the re-introduction of the civilian survivor the CLF survivor makes a return. The previous MR meant to introduce the CLF survivors. 1/9 Chance of spawning. 11% chance.

**_Engineering survivor_**

- With engineering survivor becoming more important I thought it was best to give the role better gear to help compensate for their fellow teammates weaknesses.  Engineers now spawn with 20 platesteel metal to help make the role more unique and important.

**_Scientist Survivor_**

- Having a guaranteed piece of equipment was rather lame when it came to the motion detector and the role felt a lot like a 2nd medical role. To reduce the power level and make it a little more interesting the scientist survivor now can spawn with one of 5 items that are science related.

1. Weed Killer grenade
2. EMP Grenade (Synth Killer Grenade)
3. Thermal Tarp
4. Metal Foam Grenade
5. Motion Detector

**_Botanist Survivor_**

- Removed from code and moved to the civilian survivor code reducing lines.

**_Weapon Balance Pass_**

- Prior to this update survivors (except for pmc / ert style) all received weapons from the large weapon pool. I decided to break them down into two separate weapons pool. The first pool of weapons is the more common civilian weapons a colonist might find. The other set is the security weapons which are generally better and will help the role stand out more outside from having security skills. Unlike the civilians, the security will always spawn with weapons as they are the most prepared in terms of firepower. Civilian survivors will have a 50% chance of not spawning with weapon. The previous update had it at 60% and it was rather punishing. This should help reduce the power levels of some of the more desirable survivor types. Bellow are the new weapon pools.

**_Common Weapon Pool_**

1. UZI
7. Double Barrel Shotgun
8. Hunting Rifle
9. AR-10

_**Security Survivor Weapons**_

1. M16
2. MAR-40
3. CMB Shotgun
4. MP5
5. R4T civilian rifle

**_Colony Handguns_**

- Colony handguns were removed primarily due to power creep. At the moment the colony weapons are thrown out immediately due to how useless they are. A .357 revolver for example does less damage than the marine 9mm handgun. They are deserving of a rebalance for them to be taken again and currently they just waste space.

**_Pouches_**
Since hacking skill is gone for all but engineering survivor all survivors received unique pouches to compensate for useless tools pouches. 

CL Survivor: Documents Pouch
Doctor Survivor: Medical Pouch (Randomized)
Civilian Survivor: Small pouch
Engineering & Trucker Survivor:Tools Pouch
PMC /ERT : ERT First Aid pouches

**_Survivor Skills:_**
Baseline (every surv unless overridden):
Firearms: Default (Marine)
Melee: Trained
Engineer: engineer (for hacking purposes)
Construction: Trained
Medical: Trained
Endurance: Survivor (2)

A note on the above, firearms wasn't removed, it was never a skill survs had. 

**_PMC:_**

Police: Skilled
Firearms: Trained
Doctor:

**_Medical: Doctor_**
Surgery: Trained (Doctor)

**_CLF:_**
Engineer: Engi
Medical: Medic
Surgery: Novice (Nurse)
Vehicle: Small (Squad Engi)
Scientist:

**Medical: Doctor**
Surgery: Trained (Doctor)
Research: Trained (Researcher)

**_Chef:_**
Melee Weapons: Super
Domestic: Trained

**_Miner:_**
Engineer: Engi
Powerloader: Master
Vehicle: Small (Squad Engi)

**_Trucker:_**
Engineer: Engi
Vehicle: Crewman

**_Engineer:_**
Engineer: Engineer
Construction: Engi
Powerloader: Master
Vehicle: Small (Squad Engi)

**_Chaplain:_**
Leadership: Trained (Squad Leader)

**_Marshal:_**
Police: Skilled
CQC: Skilled
Firearms: Trained

**_Prisoner:_**
Endurance: Master
CQC: Skilled
Firearms: Default (Marine)

**_Prisoner Gang Leader:_**
Endurance: Master
CQC: Skilled
Firearms: Default (Marine)
Leadership: Trained (Squad Leader)

**_Xeno Evo Times:_**
T0:

Queen: Immediate
T1:

Drone: 1 Minute
Sentinel: 4 Minutes
Runner: 5 Minutes
Defender: 4 Minutes
T2:

Hivelord: 3 Minutes
Carrier: 5 Minutes
Burrower: 7 Minutes
Spitter: 7 Minutes
Lurker: 7 Minutes
Warrior: 7 Minutes
T3:

Boiler: 15 Minutes
Praetorian: 15 Minutes
Ravager: 15 Minutes
Crusher: 15 Minutes

**_Why It's Good For The Game_**

Current round start gameplay is a mad rush to get everything perfectly in place because Xenos and survivors start off strong, this change nerfs them both and should slow down the pace.

**_Bug Fixes_**

- Scientist survivor no longer spawns with duplicate gear
- soro political prisoner survivor now spawn with winter coat again
- chaplain (nun) survivor can once again store weapon in clothing. A bad parent didn't allow for nun robes to store weapons.

Changelog
🆑
add: re-adds skill-less civilian survivor to the roster with new role types (similar to synth survivor)
add: CLF Survivor added to the roster
fix: Scientist Survivor no longer has duplicate motion detector
fix: soro political prisoner now spawns with proper winter coat
fix: chaplain survivor (nun) fixed issue with a jacket not storing weapon in previous update
fix: fixed civilian survivor code
tweak: Replaced tools pouch on survivor loadout with more role appropriate one due to loss of hacking skills
balance: Survivors now have role-specific skills, meaning one survivor is weaker, and needs allies to cover their deficiencies.
balance: More survivors spawn, with a minimum of 3 (up from 2) and a maximum of 10 (up from 8) slots. Additionally, the calculation has been changed to 1 per 9 people (up from 1 per 11).
balance: CLF ship now spawn them as a CLF synth instead of a human.
add: Added a PMC synth.
balance: Civilian Survivors now have a 50% chance not to spawn with a primary weapon
balance: Security type survivors guaranteed weapon spawn
balance: Security type survivors now draw from a different weapon pool from the rest of the civilian survivors
balance: rebalanced weapon pool aswell as made it more accurate for civilian and security survivor
balance: Survivors no longer have marine IFF.
balance: Scientist no longer has guaranteed motion detector. Now draws from a pool of science itemss.
balance: Survivors now spawn with a medium rather than a large stack of metal.
balance: Xeno castes can now only evolve after a certain amount of time has passed, generally, the more dangerous the caste, the longer it takes until they're available.
balance: Xenos now gain "bonus evolution" when they gain evolution past their evolution max. This bonus evolution is carried on to their next xeno when they evolve, meaning staying T1 for a while doesn't waste your T2 evo.
balance: Larva evo is no longer special, and works as all other xeno evo does.
/🆑

### CODE REVIEW

**Comment by Unknownity** [comment]
Remove the content inside your PR regarding synthetics as I have taken those features out and improved it on its own standalone PR #1063

**Comment by PalJohnson** [comment]
> Remove the content inside your PR regarding synthetics as I have taken those features out and improved it on its own standalone PR #1063

Will try to address issue. Thank you for bringing it up.

File: code/game/jobs/job/civilians/other/survivors.dm [hunk 1]
```diff
@@ -7,11 +7,12 @@
 	total_positions = 8
 	flags_startup_parameters = ROLE_ADD_TO_DEFAULT|ROLE_CUSTOM_SPAWN
 	late_joinable = FALSE
+	var/is_synth = FALSE
```
Comment (Unknownity) [inline code/game/jobs/job/civilians/other/survivors.dm:10]: Missed 10,27, 32-39 on removing it, the only thing you need in this survivors.dm is the ratio changes, the rest is synth related as said before

File: code/game/objects/effects/landmarks/survivor_spawner.dm [hunk 2]
```diff
@@ -1,11 +1,12 @@
 /obj/effect/landmark/survivor_spawner
 	name = "special survivor spawner"
 	var/equipment = null
+	var/synth_equipment = null
```
Comment (Unknownity) [inline code/game/objects/effects/landmarks/survivor_spawner.dm:4]: same as above, remove/revert 4,19,37

File: code/modules/gear_presets/pmc.dm [hunk 3]
```diff
@@ -634,3 +634,5 @@
 	name = "Weyland-Yutani PMC (Site Director) | HvH"
 	human_versus_human = TRUE
 	headset_type = /obj/item/device/radio/headset/distress/PMC/hvh/cct
+
+//*****************************************************************************************************/
```
Comment (Unknownity) [inline code/modules/gear_presets/pmc.dm:638]: Forgot to remove it entirely? two unnecessary extra lines

**Review by Unknownity — COMMENTED** [review]
you missed some stuff, adressed in review comments (they should appear)

**Comment by Watermelon914** [comment]
fix up your title, it's not detailed enough. The title should be able to tell me what the PR is about, tweak doesn't tell me anything.

**Comment by PalJohnson** [comment]
> fix up your title, it's not detailed enough. The title should be able to tell me what the PR is about, tweak doesn't tell me anything.

adressed

**Comment by PalJohnson** [comment]
> you missed some stuff, adressed in review comments (they should appear)

applied as requested. Thanks

**Comment by fira** [comment]
code LGTM

**Review by fira — APPROVED** [review]

**Comment by PalJohnson** [comment]
https://github.com/cmss13-devs/cmss13/pull/1586

refer to new PR

**Pull request closed without merging**